tools/Dockerfile

```diff
@@ -18,6 +18,10 @@ ARG HELMFILE_VERSION=v0.128.1
 ARG EKSCTL_VERSION=0.30.0
```

Comment:
please make sure you don't revert anything in the version of this file on master, as it has some more updates there